static/app/components/events/featureFlags/featureFlagInlineCTA.tsx

```diff
@@ -0,0 +1,167 @@
+    <InterimSection
+      help={t(
+        "The last 100 flags evaluated in the user's session leading up to this event."
+      )}
+      isHelpHoverable
+      title={t('Feature Flags')}
+      type={SectionKey.FEATURE_FLAGS}
+      actions={actions}
+    >
+      <BannerWrapper>
+        <div>
+          <BannerTitle>{t('Set Up Feature Flags')}</BannerTitle>
+          <BannerDescription>
+            {t(
+              'Want to know which feature flags were associated with this error? Set up your feature flag integration.'
+            )}
+          </BannerDescription>
+          <ActionButton>
+            <Button onClick={handleSetupButtonClick} priority="primary">
+              {t('Set Up Now')}
+            </Button>
+            <LinkButton
+              priority="default"
+              href="https://docs.sentry.io/product/issues/issue-details/#feature-flags"
+              external
+            >
+              {t('Read More')}
+            </LinkButton>
+          </ActionButton>
+        </div>
+        <CloseDropdownMenu
+          position="bottom-end"
+          triggerProps={{
+            showChevron: false,
+            borderless: true,
+            icon: <IconClose color="subText" />,
+          }}
+          size="xs"
+          items={[
+            {
+              key: 'dismiss',
+              label: t('Dismiss'),
+              onAction: () => {
+                dismissPrompt();
+                trackAnalytics('flags.cta_dismissed', {
+                  organization,
+                  type: 'dismiss',
+                });
+              },
+            },
+            {
+              key: 'snooze',
+              label: t('Snooze'),
+              onAction: () => {
+                snoozePrompt();
+                trackAnalytics('flags.cta_dismissed', {
+                  organization,
+                  type: 'snooze',
+                });
+              },
+            },
+          ]}
+        />
+      </BannerWrapper>
+    </InterimSection>
+  );
+}
+
+const BannerTitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  margin-bottom: ${space(1)};
+  font-weight: ${p => p.theme.fontWeightBold};
+`;
+
+const BannerDescription = styled('div')`
+  margin-bottom: ${space(1.5)};
+  max-width: 340px;
+`;
+
+const CloseDropdownMenu = styled(DropdownMenu)`
+  position: absolute;
+  display: block;
+  top: ${space(1)};
+  right: ${space(1)};
+  color: ${p => p.theme.white};
+  cursor: pointer;
+  z-index: 1;
+`;
+
+const ActionButton = styled('div')`
+  display: flex;
+  gap: ${space(1)};
+`;
+
+const BannerWrapper = styled('div')`
+  position: relative;
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+  padding: ${space(2)};
+  margin: ${space(1)} 0;
+  background: linear-gradient(
+    90deg,
+    ${p => p.theme.backgroundSecondary}00 0%,
+    ${p => p.theme.backgroundSecondary}FF 70%,
+    ${p => p.theme.backgroundSecondary}FF 100%
+  );
+`;
```

Comment:
new type/style of banner with new css... this is like the 4th type we have?

cc @Jesse-Box

Reply:
looks like it snaps together quick tho

Reply:
actually this is reused from `static/app/components/events/eventReplay/replayInlineOnboardingPanel.tsx`. it's the same style as the replay inline banner on issue details

https://github.com/getsentry/sentry/blob/8be9591d9d21b3abda56868a978ae8c166d78b2e/static/app/components/events/eventReplay/replayInlineOnboardingPanel.tsx#L116